<a href="https://colab.research.google.com/github/jinzhouyang666/How-Language-Shapes-AI-Generated-Political-Discourse-A-Prototype-Analysis/blob/main/Prototype_Analysis_AI_Responses_to_Political_Corruption_Prompts.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# 🎯 How Language Shapes AI-Generated Political Discourse: A Prototype Analysis in English and Dutch

With the rapid rise of **Generative AI (GenAI)** tools like ChatGPT, these technologies are increasingly integrated into everyday life as a medium for **political inquiry, deliberation, and even companionship**. While much of the academic focus has centered on **algorithmic bias** and the **disinformation potential** of large language models (LLMs), very little is known about how people **actually engage with these tools to discuss political topics**, and how such interactions may influence their political attitudes.

Recent research presents a paradox: GenAI has the potential to **reduce belief in conspiracy theories** and **facilitate dialogue across partisan divides**, yet it may also **reinforce existing opinions** and subtly shape user worldviews without transparency. This experimental project responds to that.

We investigate a underexplored factor:  
> Does the **language** in which a political question is asked shape how GenAI (specifically ChatGPT) frames its answer, emotionally, morally, or civically?

In other words:
- Would a Dutch-speaking user receive a **more neutral, system-trusting answer** to the same political prompt than an English-speaking user?
- Does the model show different degrees of **empathy, encouragement, or skepticism** across languages?
- Could such subtle differences, repeated at scale, affect democratic discourse?


## Experimental Design

To explore this, we designed a **comparative prompt-response experiment** across two languages:

- 7 political prompts were crafted in both **English and Dutch**, based on confirmed corruption-related news events.
- Prompts were categorized into four thematic areas:
  - A – Public trust and systemic framing
  - B – Civic action and motivation
  - C – AI as a political risk
  - D – Cultural approaches to corruption
- We then queried **GPT-4 Omni**, using default settings, and recorded the AI’s responses in both languages.

Each response was analyzed using:
- **TextBlob** for English sentiment (polarity & subjectivity)
- **DutchSentimentAnalysis** for Dutch sentiment scoring and labeling


This pilot study serves as a **computational text analysis precursor** to the broader doctoral research agenda:  
> How do GenAI conversations influence political opinion formation, toward **deliberation** or **polarization**?

By showing that the language of interaction alone may influence how GenAI frames democratic issues, this work raises:
- Does GenAI unintentionally modulate political tone across linguistic-cultural boundaries?
- Could such framing affect trust in institutions, perceived agency, or civic willingness to act?

These findings will contribute to understanding the **opportunities and democratic risks of GenAI**.


## Step 1: Prompt and Answer Dataset

### Data Context
The prompts used in this experiment were **designed based on real corruption news events** reported in [May 2025](https://www.transparency.nl/nieuws/2025/05/corruptieschandaal-bij-navo-3-nederlanders-opgepakt/), including:
- NATO procurement-related arrests in the Netherlands
- Reports of corruption in defense contracts contributing to arms trafficking
- The [misuse of AI](https://www.transparency.nl/nieuws/2025/04/naast-kansen-ook-veel-risicos-bij-ai-op-het-gebied-van-corruptie/) in political disinformation and democratic manipulation

All scenarios assume **confirmed or credible corruption events**, ensuring that the prompts do not speculate but reflect actual issues in public discourse.

### Prompt Construction
Each prompt was carefully formulated in **both English and Dutch**, and fed into **ChatGPT-4.0** to generate a neutral, full-length answer. The prompts were divided into four **thematic categories**:

| Category | Meaning |
|----------|---------|
| **A – Systemic Trust** | How people perceive corruption within institutions (e.g., "Is corruption inevitable?") |
| **B – Civic Action** | Whether citizens feel empowered to act against corruption (e.g., "What can we do?") |
| **C – AI Risk** | Concerns about AI being misused for corruption or disinformation |
| **D – Cultural Framing** | Whether corruption is treated differently across political cultures |

Each category contains two prompts (one in each language) for controlled comparison.

### Output Content
- **Prompt**: User-initiated question (English or Dutch)
- **Answer**: GPT-4 Omni's response in the same language
- All responses were generated using default ChatGPT behavior (no system prompt modification).

The following are shown in English and Dutch respectively.


In [3]:
import pandas as pd
from IPython.display import display, HTML

file_path = "/content/Prompt and  Answer sheet - Sheet1.csv"
df = pd.read_csv(file_path)

df['Category'] = df['Category'].fillna(method='ffill')

df['Answer_Length'] = df['Answer'].str.len()
df['Word_Count'] = df['Answer'].str.split().apply(len)
df['Sentence_Count'] = df['Answer'].str.count(r'[.!?]')

# Split by language
df_en = df[df['Language'] == 'English'].reset_index(drop=True)
df_nl = df[df['Language'] == 'Dutch'].reset_index(drop=True)

# Display the first few English responses
display_columns = ['Prompt_ID', 'Category', 'Prompt', 'Answer']
display(HTML(df_en[display_columns].to_html(index=False)))


<ipython-input-3-1a1c163aec83>:7: FutureWarning: Series.fillna with 'method' is deprecated and will raise in a future version. Use obj.ffill() or obj.bfill() instead.
  df['Category'] = df['Category'].fillna(method='ffill')


In [9]:
# Display Dutch responses (total: 7 entries)
df_nl_display = df_nl[['Prompt_ID', 'Category', 'Prompt', 'Answer']]
df_nl_display.style.hide(axis='index')


### Step 2: Sentiment Analysis of English Responses

### Objective:
This step aims to explore how ChatGPT frames political corruption issues **in emotional terms** in its English responses. Specifically, we are interested in whether the answers:

- Adopt a more **positive or negative tone**,
- Are **subjective (opinion-based)** or **objective (fact-based)**,
- Differ **between prompt types** (e.g., "Is corruption inevitable?" vs. "What can citizens do?"),
- Show **inconsistencies across individual prompts** (A1, A3, etc.), even within the same category.

Language framing plays a crucial role in shaping users' perceptions. Previous research (e.g., Lakoff, 2004; Entman, 1993) shows that **how issues are linguistically framed** affects civic attitudes, motivation, and trust.

By applying **sentiment analysis**, we measure:
- The **emotional valence** of ChatGPT's answers (negative vs. positive),
- The **degree of subjectivity** (factual vs. opinionated framing),
These are useful proxies for evaluating **affective framing** and **epistemic positioning**.

### Method:
We use [**TextBlob**](https://textblob.readthedocs.io/en/dev/), a standard Python NLP library, to analyze the sentiment of each answer. For each response, we compute:

- **Polarity**  
  Numerical value between -1.0 and 1.0  
  - **> 0**: positive tone  
  - **< 0**: negative tone  
  - **≈ 0**: neutral or mixed

- **Subjectivity**  
  Numerical value between 0.0 and 1.0  
  - **≈ 0**: more factual, analytical  
  - **≈ 1**: more opinionated, personal

These scores help identify whether ChatGPT:
- Promotes civic engagement (often framed positively),
- Emphasizes helplessness or system failure (often framed negatively),
- Takes a neutral informational stance.

While each prompt (A1, A3, etc.) is unique, they are grouped under broader categories that reflect **thematic intent**. By aggregating polarity and subjectivity within each category, we can:
- Compare overall **framing tendencies** (e.g., “Does ChatGPT frame civic action prompts more optimistically than systemic corruption prompts?”)
- Control for prompt-level noise and interpret **category-level patterns**

**However**, we also keep **individual prompt scores** (e.g., A1, A3) to:
- Identify **outliers or inconsistencies**
- Ensure **micro-level interpretability**

This dual approach (aggregate + individual) balances interpretability and structure.


In the next step, we will visualize:
- Sentiment scores by category (mean polarity and subjectivity),
- Individual scores per prompt (A1, A3, etc.) for micro-level differences.


In [36]:
!pip install textblob
from textblob import TextBlob

# Analyze sentiment for each English response
df_en['Polarity'] = df_en['Answer'].apply(lambda x: TextBlob(x).sentiment.polarity)
df_en['Subjectivity'] = df_en['Answer'].apply(lambda x: TextBlob(x).sentiment.subjectivity)

# Display results for each prompt (hide index)
df_en_display = df_en[['Prompt_ID', 'Category', 'Polarity', 'Subjectivity']]
df_en_display.style.hide(axis='index')



## Step 2.2 Analysis of Sentiment Scores (English Answers)

We analyzed the emotional framing of ChatGPT's responses in English using TextBlob. Each response was scored across two dimensions:

- **Polarity (−1 to +1):**  
  Measures emotional **tone**:
  - `> 0`: Positive language (hopeful, constructive)
  - `< 0`: Negative language (frustrated, skeptical)
  - `≈ 0`: Neutral, mixed, or analytical tone

- **Subjectivity (0 to 1):**  
  Measures **opinionated vs factual** tone:
  - `0`: Fully objective (factual, data-driven)
  - `1`: Fully subjective (personal, belief-based)


### Results Table

| Prompt_ID | Category | Polarity | Subjectivity | Interpretation |
|-----------|----------|----------|--------------|----------------|
| **A1** | A | 0.097 | 0.577 | Mildly positive and moderately subjective. ChatGPT frames corruption as solvable through international cooperation. |
| **A3** | A | 0.082 | 0.631 | Similar tone to A1 — hopeful yet not assertive. Slightly more opinion-based. |
| **B1** | B | -0.127 | 0.413 | Negative polarity suggests a sense of helplessness. Moderately factual tone. |
| **B3** | B | 0.039 | 0.262 | Neutral to slightly positive tone, but strongly objective — ChatGPT offers civic advice without emotional push. |
| **C1** | C | -0.162 | 0.696 | Clearly negative tone with high subjectivity. Reflects emotional concern over AI misuse. |
| **C3** | C | 0.042 | 0.589 | Slightly positive, still subjective — possibly downplaying deepfake risks with reassurance framing. |
| **D1** | D | 0.115 | 0.296 | Most positive and also one of the most objective — reflects neutral cultural comparison language. |

---

### Takeaways

1. **Corruption framing (A):**  
   ChatGPT uses **softly optimistic** and **moderately opinionated** language, suggesting reform is possible, but not forcefully encouraging action.

2. **Civic action prompts (B):**  
   Mixed polarity: B1 is emotionally discouraged, while B3 is **informational**, ChatGPT avoids emotional commitment.

3. **AI risk framing (C):**  
   **C1 stands out** with the **most negative and subjective** tone, likely due to the inherent fear narrative around AI misuse.  
   C3 softens the message, possibly reflecting AI's bias towards balanced language.

4. **Cultural framing (D):**  
   Polarity is **most positive**, subjectivity **lowest**, suggesting a preference for **neutral, cross-cultural explanations**.

- ChatGPT avoids extreme polarity, and none of the responses are > 0.2 or < -0.2. This may reflect its safety and moderation constraints.
- Subjectivity is higher for emotionally loaded issues (e.g., AI threats), and lower in prompts asking for civic procedure or comparisons.


## Step 3: Sentiment Analysis of Dutch Responses

We conduct sentiment analysis directly on the original Dutch texts. This preserves:
- Cultural nuances
- Idiomatic expressions
- Authentic emotional framing

### Method

We use the [**DutchSentimentAnalysis** package](https://github.com/Glender/DutchSentimentAnalysis) by Glenn Hiemstra, which:
- Uses a Dutch sentiment dictionary with ~5000 annotated words
- Scores sentiment from **-2 (very negative)** to **+2 (very positive)**
- Accounts for negations (e.g., "niet goed")
- Demonstrates high predictive validity (r = 0.75) in real-world datasets

This provides a reliable, language-appropriate method to evaluate ChatGPT's emotional framing in Dutch.




In [11]:

!pip install -q rpy2


In [14]:

!apt-get install -y r-base


Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
r-base is already the newest version (4.5.0-3.2204.0).
0 upgraded, 0 newly installed, 0 to remove and 34 not upgraded.


In [18]:
%load_ext rpy2.ipython


The rpy2.ipython extension is already loaded. To reload it, use:
  %reload_ext rpy2.ipython


In [17]:
%%R
# Install dependencies
install.packages("devtools")
install.packages("remotes")
remotes::install_url("https://raw.githubusercontent.com/Glender/DutchSentimentAnalysis/main/inst/script/vwr_0.3.0.tar.gz")
devtools::install_version("stringr", version = "1.4.0", repos = "http://cran.us.r-project.org")
devtools::install_github("Glender/DutchSentimentAnalysis")


── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘/tmp/Rtmpr1kF4Z/remotes1e5628f69a9/vwr/DESCRIPTION’ ... OK
* preparing ‘vwr’:
* checking DESCRIPTION meta-information ... OK
* checking for LF line-endings in source and make files and shell scripts
* checking for empty or unneeded directories
* building ‘vwr_0.3.0.tar.gz’

These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                            
2: CRAN packages only             
3: None                           
4: stringr (1.4.0 -> 1.5.1) [CRAN]

Enter one or more numbers, or an empty line to skip updates: 
── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘/tmp/Rtmpr1kF4Z/remotes1e525986061/Glender-DutchSentimentAnalysis-86c8198/DESCRIPTION’ ... OK
* preparing ‘DutchSentimentAnalysis’:
* checking DESCRIPTION meta-information ... OK
* checking fo

Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
trying URL 'https://cran.rstudio.com/src/contrib/devtools_2.4.5.tar.gz'
Content type 'application/x-gzip' length 374718 bytes (365 KB)
downloaded 365 KB


The downloaded source packages are in
	‘/tmp/Rtmpr1kF4Z/downloaded_packages’
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
trying URL 'https://cran.rstudio.com/src/contrib/remotes_2.5.0.tar.gz'
Content type 'application/x-gzip' length 164496 bytes (160 KB)
downloaded 160 KB


The downloaded source packages are in
	‘/tmp/Rtmpr1kF4Z/downloaded_packages’
Installing 2 packages: interp, latticeExtra
Installing packages into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
trying URL 'https://cran.rstudio.com/src/contrib/interp_1.1-6.tar.gz'
trying URL 'https://cran.rstudio.com/src/contrib/latticeExtra_0.6-30.tar.gz'

The downloaded source packages are in
	‘/tmp/Rtmpr1kF4Z/downloaded_packages’
Installing package in

In [19]:
answers_nl = df_nl['Answer'].tolist()


In [20]:
import rpy2.robjects as robjects
from rpy2.robjects import StrVector

robjects.globalenv['text'] = StrVector(answers_nl)


In [21]:
%%R
library(DutchSentimentAnalysis)
library(tibble)

results <- tibble(
  Answer = text,
  Score = dutch_sentiment_analysis(text),
  Label = dutch_sentiment_analysis(text, output = "label")
)

results


# A tibble: 7 × 3
  Answer                                                             Score Label
  <chr>                                                              <dbl> <chr>
1 "Corruptie in de internationale politiek is een complex en veel…  0.0256 neut…
2 "Je zorgen zijn begrijpelijk, vooral gezien de complexiteit en …  0.633  posi…
3 "Het is begrijpelijk dat je je machteloos voelt, maar er zijn m…  0.433  posi…
4 "In een democratie kunnen burgers verschillende stappen onderne… -0.157  neut…
5 "AI kan zowel positieve als negatieve effecten hebben op corrup… -0.0784 neut…
6 "Ja, de opkomst van deepfakes en AI-bots in de politiek is zorg…  0.111  neut…
7 "Ja, er zijn verschillen in hoe politieke corruptie wordt aange…  0.0244 neut…


Emotions do not lie


In [24]:
from rpy2.robjects import pandas2ri
from rpy2.robjects.packages import importr
from rpy2.robjects import r
import rpy2.robjects as robjects

pandas2ri.activate()



In [25]:

results_r_df = robjects.r['results']

df_results_nl = pandas2ri.rpy2py(results_r_df)


In [27]:

df_nl = df_nl.reset_index(drop=True)

df_nl['NL_Score'] = df_results_nl['Score'].values
df_nl['NL_Label'] = df_results_nl['Label'].values

df_nl_display = df_nl[['Prompt_ID', 'Category', 'NL_Score', 'NL_Label','Prompt', 'Answer']]
df_nl_display.style.hide(axis='index')


In [33]:

df_nl_summary = df_nl[['Prompt_ID', 'Category', 'NL_Score', 'NL_Label']]

df_nl_summary = df_nl_summary.sort_values(by=['Category', 'Prompt_ID']).reset_index(drop=True)

df_nl_summary.style.hide(axis='index')


## Step 3.1: Interpretation of Sentiment Analysis on Dutch Responses

We now interpret the sentiment scores (`NL_Score`) and labels (`NL_Label`) obtained from the **DutchSentimentAnalysis** package.

The package uses a **Dutch sentiment dictionary**, each assigned a numeric score from **-2 to +2**:
- `+2`: strongly positive
- `+1`: positive
- ` 0`: neutral
- `-1`: negative
- `-2`: strongly negative

The algorithm:
- Matches input text against the dictionary
- Adjusts for **negations** (e.g. "niet goed")
- Averages across all recognized terms in the input
- Outputs a final score (e.g., 0.63 = moderately positive)

Each final score is then mapped to a sentiment label:
- **positive**: score > 0.2
- **neutral**: score between -0.2 and 0.2
- **negative**: score < -0.2

### Result
- **Category A (Corruption trust framing):**
  - A2: Mostly neutral (institutional limits, no strong stance)
  - A4: Clearly positive (constructive tone about transparency and reform)

- **Category B (Civic engagement framing):**
  - B2: Positive (encourages action)
  - B4: Slightly negative, but not strongly enough to be labeled negative

- **Category C (AI risk framing):**
  - Both prompts yield scores close to zero, ChatGPT takes a measured, cautious tone

- **Category D (Cultural comparison):**
  - Extremely neutral, the language is analytical, descriptive, no emotional charge


These scores suggest that:
- ChatGPT in Dutch maintains a **cautious and reserved emotional tone** on corruption-related issues.
- Even positive responses (e.g., B2) stay within moderate emotional bounds.
- No **explicitly negative** tone appears, indicating a diplomatic or institutional alignment in tone.

This might reflect:
- ChatGPT’s design to minimize emotional language in sensitive policy topics
- A possible language calibration difference (Dutch language model is more formal, or training data is more bureaucratic)


In [34]:

df_en_summary = df_en[['Prompt_ID', 'Category', 'Polarity', 'Subjectivity']]

df_en_summary = df_en_summary.sort_values(by=['Category', 'Prompt_ID']).reset_index(drop=True)

df_en_summary.style.hide(axis='index')


In [35]:

df_en_clean = df_en[['Prompt_ID', 'Category', 'Polarity', 'Subjectivity']].copy()
df_en_clean.rename(columns={'Polarity': 'Score'}, inplace=True)
df_en_clean['Sentiment_Label'] = None
df_en_clean['Language'] = 'English'

df_nl_clean = df_nl[['Prompt_ID', 'Category', 'NL_Score', 'NL_Label']].copy()
df_nl_clean.rename(columns={'NL_Score': 'Score', 'NL_Label': 'Sentiment_Label'}, inplace=True)
df_nl_clean['Subjectivity'] = None
df_nl_clean['Language'] = 'Dutch'


df_combined = pd.concat([df_en_clean, df_nl_clean], ignore_index=True)

df_combined = df_combined.sort_values(by=['Category', 'Prompt_ID']).reset_index(drop=True)

df_combined_display = df_combined[['Prompt_ID', 'Category', 'Language', 'Score', 'Subjectivity', 'Sentiment_Label']]
df_combined_display.style.hide(axis='index')


<ipython-input-35-c7d9cb348e63>:14: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
  df_combined = pd.concat([df_en_clean, df_nl_clean], ignore_index=True)


## Step 4 Sentiment Comparison Between English and Dutch Responses

This section presents a sentiment analysis of ChatGPT's responses to identical prompts presented in **English** and **Dutch**, across four thematic categories, and each response is evaluated by:
- **Polarity score**
- **Subjectivity**
- **Sentiment label**: based on Dutch classification (negative, neutral, positive)


### A. Corruption and Trust in Institutions

- **Observation**: A4 (Dutch) is significantly more positive (+0.633) than A3 (English, +0.082).
- **Interpretation**: While both A1/A2 show neutral or mildly optimistic tone, A4 adopts a far more empathetic and mobilizing tone ("Je zorgen zijn begrijpelijk").
- This suggests **ChatGPT’s Dutch responses may offer more emotional reassurance**, especially when the user expresses disillusionment with political institutions.


###  B. Civic Action and Empowerment

- **B1 vs B2**: A sharp contrast in tone, English response to civic helplessness is **negative** (-0.127), whereas Dutch is **strongly positive** (+0.433).
- **Language Use**:
  - Dutch B2 includes direct action language like "je kunt iets doen", signaling empowerment.
  - English B1 centers emotional powerlessness ("Feeling powerless…"), with more hesitant suggestions.
- **B3 vs B4**: While English B3 is mildly positive, Dutch B4 dips to -0.157, more descriptive and less motivating.

This reflects ChatGPT’s Dutch responses can be **more affirming and activating**, but also more **cautious in factual lists** (B4). English tends to be more emotionally flat or uncertain.


### C. AI as Risk/Tool for Corruption

- English responses carry **more emotional load**, with one showing strongly negative framing (C1, -0.162) and high subjectivity (0.696).
- Dutch outputs are **more neutral and factual**, consistent with system-focused tone in Dutch-language civic discourse.
- **Framing Difference**:
  - English: “deepfakes are worrying” / “threaten public trust”
  - Dutch: emphasizes regulation, transparency, and shared responsibility

**ChatGPT in Dutch seems to lean into systemic frames**, avoiding emotionally loaded language even on high-risk topics like AI disinformation.


### D. Cultural Differences in Corruption

- Both answers are **moderately positive**, but the Dutch version is less emotionally expressive.
- English output includes mentions of "strong regulations" and praise of EU transparency.
- Dutch output shifts focus to factual comparisons between EU institutions and international norms.

ChatGPT answers about cultural variation are **less affective overall**, but Dutch responses still tend to be **more reserved**.


## Interpretation

- **Language Matters**: ChatGPT produces measurably different answers not just translation differences, but **framing, tone, and emotional valence**.
- **Dutch Responses**:
  - More likely to be neutral (5 out of 7 scored as "neutral")
  - Tend to emphasize **system-level solutions** and **procedural language**
  - Occasionally more **empowering** (e.g., B2) or **empathic** (e.g., A4)
- **English Responses**:
  - Exhibit higher **emotional range** (positive and negative polarity)
  - Contain more **subjective cues**, especially in prompts expressing concern or helplessness

## Conclusion

These findings support our hypothesis:

> **Generative AI adapts its political framing strategies based on language and cultural context**, which can shape how users interpret issues of corruption, trust, and civic participation.

This polarity divergence across identical prompts in English and Dutch demonstrates that:
- **Framing is not uniform**, even from the same model.
- **Civic mobilization potential** varies with language.



## Step 5 Limitations and Future Work

While this experiment provides initial insights into how ChatGPT frames political corruption across languages, several important limitations should be acknowledged:

### 1. Prompt framing bias
All prompts were constructed from a **trusting position toward the corruption allegations**, as they were derived from recent Dutch news reports. As such, the prompts assumed that the corruption claims were real and serious. This framing may have constrained ChatGPT’s possible responses, encouraging alignment rather than allowing for skeptical, defensive, or critical perspectives. Future iterations should include **prompts that express doubt or challenge the corruption narrative**. This would help explore whether GenAI aligns with dominant narratives or balances competing perspectives.

### 2. Dutch sentiment analysis limitations
The Dutch sentiment analysis was conducted using the **DutchSentimentAnalysis package**, which is based on a fixed sentiment lexicon of ~5,000 words. While it accounts for negations and performs well in product review contexts, it may struggle with:
- **Subtle political hedging**, e.g., "niet helemaal transparant" (not fully transparent)
- **Nuanced legal/administrative language**
- **Emotionally neutral but ideologically loaded terms**, such as "verantwoordelijkheid" (responsibility)

In contrast to the more mature tools available for English (e.g.,BERT-based classifiers), the Dutch tool may underdetect emotional polarity or over-rely on lexical features. Future analysis could benefit from **context-aware models or human-coded validation**.

### 3. Lack of situational context
This experiment analyzed **isolated Q&A prompts**, without simulating realistic interactional dynamics. In real-world usage, users bring:
- Personal stakes
- Prior beliefs
- Emotional tone in multi-turn conversations

This context is **crucial to understanding how GenAI might reinforce, challenge, or reshape political beliefs**. Our current analysis does not account for these dynamics. Therefore, results should be interpreted as preliminary insight into **static framing differences**, not dynamic influence.


## Future Work

This exploratory study demonstrates that **linguistic context meaningfully affects how GenAI frames political issues**. As a foundation, it points toward several research directions that align with the goals of the broader project:

- **Survey research**: Conduct a representative user survey to explore how people from different language/cultural backgrounds use GenAI to discuss political issues, and how much they trust its answers.
  
- **Experimental research**: Build on the polarity differences found here to design an experiment where users are randomly assigned to see different GenAI responses and then report:
  - Perceived fairness or bias
  - Agreement with content
  - Willingness to act (e.g., sign a petition, share online)

- **Data donation and dialogue analysis**: Extend beyond single-turn prompts to collect real-world, multi-turn GenAI political conversations through data donation. This would allow us to analyze:
  - Escalation or de-escalation of political affect
  - How user framing and AI responses co-evolve

Together, these steps would allow us to better understand the project. This pilot study suggests that **language alone may shape perceived neutrality or bias**, which has deep implications for how people use and interpret GenAI in democratic contexts.
